src/App.jsx

```diff
@@ -1,6 +1,28 @@
+import { useState } from "react"
+import { useEffect } from "react"
+import Card from "./components/Card"
+import data from "./data"
+
 const App = () => {
+  const [count, setCount] = useState(0)
+  const [usersData, setUsersData] = useState([])
+  useEffect(()=>{
+    async function getData(){
+      try {
+        const res = await fetch("https://random-data-api.com/api/users/random_user?size=10")
+        const data = await res.json()
+        let array = []
+        data.forEach(user=> array.push({image: user.avatar, fname : user.first_name, lname : user.last_name, employment:user.employment}))
+        console.log(array);
```

Comment:
To improve the quality of the code, please consider refactoring it by removing the console.log() statement